src/services/firebase.ts

```diff
@@ -112,7 +112,29 @@ export namespace ourFirebase {
 
   // TODO: export function createMatch(game: GameInfo): MatchInfo {}
   // TODO: export function addParticipant(match: MatchInfo, user: User) {}
-  // TODO: export function updateMatchState(match: MatchInfo, matchState: MatchState) {}
+  export function updateMatchState(match: MatchInfo, matchState: MatchState) {
+    const newState: fbr.CurrentState = {};
+    for (let item in matchState) {
+      console.log(item);
+      newState[item] = {
+        y: matchState[item].y,
+        zDepth: matchState[item].zDepth,
+        currentImageIndex: matchState[item].currentImageIndex,
+        cardVisibility: matchState[item].cardVisibility,
+        rotationDegrees: 360,
+        drawing: {}
+      };
+    }
+
+    const updates: any = {};
+    updates[
+      `gamePortal/gamePortalUsers/matches/${match.matchId}/pieces`
```

Comment:
Usually, for better readability, split it in two lines, one for defining variable and other for using that variable as index value for array.